Documentation/data-collection.md

```diff
@@ -560,6 +560,15 @@ data:
     # 99th percentile of the backend commit duration.
     - '{__name__="instance:etcd_disk_backend_commit_duration_seconds:histogram_quantile",quantile="0.99"}'
     #
+    # Number of jaeger instances using certain storage type.
```

Comment:
Could you add the name of the team that owns these metrics in the same format as the other metrics?

Reply:
What format the team name should have? The team own this metrics will be `@tracing-team`  or is at least the slack name of the team.